source/_integrations/updater.markdown

```diff
@@ -16,6 +16,8 @@ ha_platforms:
 <div class='note warning'>
 
 This integration is deprecated and will be removed in Home Assistant Core 2022.5.
+  
+Similar functionality is available using the [Version integration](/integrations/version/).
```

Comment:
This is only partially true.
What Version integration provides is only part of what the updater integration did.
The Analytics integration has taken over the reporting part of the updater integration.

Reply:
That is the only part there for quite some time, and it is what is causing the breaking change at this point. So, I guess this is fine/enough to have at this point.

Reply:
To be honest I wasn't sure how much detail to put in to that. Happy to add more around that if needed/wanted but I'll admit I'd be copy/pasting from the release notes mentioning it being removed given I don't know the context of the analytics part